unicorn_mode/build_unicorn_support.sh

```diff
@@ -27,19 +27,16 @@
 # You must make sure that Unicorn Engine is not already installed before
 # running this script. If it is, please uninstall it first.
 
-UNICORN_URL="https://github.com/unicorn-engine/unicorn/archive/1.0.1.tar.gz"
-UNICORN_SHA384="489f2e8d18b6be01f2975f5128c290ca0c6aa3107ac317b9b549786a0946978469683e8fa8b6dfc502f6f71242279b47"
-
 echo "================================================="
 echo "Unicorn-AFL build script"
 echo "================================================="
 echo
 
 echo "[*] Performing basic sanity checks..."
 
-if [ "$(id -u)" != "0" ]; then
+if [ "$(id -u)" = "0" ]; then
```

Comment:
Should we specifically preclude users from installing as root here? Would it be better to just remove this check completely?

Reply:
I can do that, works for me.